testsuite/scripts/tutorials/test_01-lennard_jones.py

```diff
@@ -28,9 +28,13 @@
 class Tutorial(ut.TestCase):
     system = tutorial.system
 
-    def test_rdf_curve(self):
-        self.assertGreater(
-            np.corrcoef(tutorial.rdf, tutorial.theo_rdf)[1, 0], 0.985)
+    def test_rdf(self):
+        rms = np.sqrt(((tutorial.rdf - tutorial.theo_rdf)**2).mean()) / len(tutorial.r)
```

Comment:
* why divide by the length of the data series outside the square root?
* would it be more helpful to compare the data series directly? e.g. with `np.testing.assert_allclose(tutorial.rdf, tutorial.theo_rdf, rtol=0, atol=5e-4)`, in case of failure the test would format the two data series and give you a summary of the statistics (number of mismatched data points, max relative difference, max absolute difference). This would however make the test a lot more sensitive to outliers, wherease the RMS averages out artifiacts.

Reply:
The idea was that the rms obviously depends on the number of data points, to get an independent measure one could calculate the RMS per data point. However, I agree usnig `np.testing.assert_allclose(tutorial.rdf, tutorial.theo_rdf, rtol=0, atol=5e-4)` is much more elegant.